lsp-completion.el

```diff
@@ -731,8 +728,13 @@ The CLEANUP-FN will be called to cleanup."
      (lsp-completion-mode
       (setq-local completion-at-point-functions nil)
       (add-hook 'completion-at-point-functions #'lsp-completion-at-point nil t)
-      (setq-local completion-category-defaults
-                  (add-to-list 'completion-category-defaults '(lsp-capf (styles basic))))
+      (make-local-variable 'completion-category-defaults)
+      (setf (alist-get 'lsp-capf completion-category-defaults) '((styles . (pass-through))))
+      (make-local-variable 'completion-styles-alist)
+      (setf (alist-get 'pass-through completion-styles-alist)
```

Comment:
I would call the style `lsp-passthrough`, not `pass-through` in order to make the provenance of the completion style clear and in order to avoid name clashes.